config/release.js

```diff
@@ -0,0 +1,27 @@
+/* jshint node:true */
+var RSVP = require('rsvp');
+var exec = require('child_process').exec;
+
+module.exports = {
+  publish: true,
+  afterPublish: function(project, versions) {
+    // Publish dummy app with docs to gh-pages
+    return runCommand('ember github-pages:commit --message "Released ' + versions.next + '"').then(function() {
```

Comment:
Since you control the system you are running on, I would suggest using `execSync` instead of `exec` here...

```js
var execSync = require('child_process').execSync;

function runCommand(command) {
  console.log('running: ' + command);
  var output = execSync(command, { encoding: 'utf8' });
  console.log(output);
}

module.exports = {
  publish: true,
  afterPublish: function(project, versions) {
    runCommand('ember github-pages:commit --message "Released ' + versions.next + '"');
    runCommand('git push origin gh-pages:gh-pages');
  }
};
```